app/src/main/java/cellscanner/wowtor/github/com/cellscanner/BootDeviceReceiver.java

```diff
@@ -25,8 +25,13 @@ public void onReceive(Context context, Intent intent) {
 
         // TODO: @ check if the right application is sending the request
         if (Recorder.inRecordingState(context)){
-            // TODO: make startup version aware
-            LocationService.start(context);
+            // TODO: Move this logic to the Recorder Class
+            // The recorder class should decide to run the service as Android 8 + or not
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
Kunnen we dit niet op één plek afhandelen? Zowel `Recorder.startService` als `LocationService.start` zijn onze eigen static functies, dus dat onderscheid >=Oreo en <Oreo zouden we ook daarbinnen kunnen maken.

Reply:
Correct, uiteindelijk moet er maar 1 functie zijn die wordt aangeroepen door de verschillende entry points. Maar het is mijn leerprocess dat daar langzaam naartoe werkt.

Reply:
Mijn planning is een volgend pr waaring de intergratie volledig is. Dit PR is bedoeld om de LVL 26+ API telaten werken. De volgende zal het geheel verbinden